liberty-maven-plugin/src/main/java/io/openliberty/tools/maven/server/GenerateFeaturesMojo.java

```diff
@@ -88,14 +100,6 @@ private void generateFeatures() throws PluginExecutionException {
             log.debug("Generate features for all class files");
         }
 
-        Map<String, File> libertyDirPropertyFiles;
-        try {
-            libertyDirPropertyFiles = BasicSupport.getLibertyDirectoryPropertyFiles(installDirectory, userDirectory, serverDirectory);
-        } catch (IOException e) {
-            log.debug("Exception reading the server property files", e);
-            log.error("Error attempting to generate server feature list. Ensure your user account has read permission to the property files in the server installation directory.");
-            return;
-        }
```

Comment:
@turkeylurkey Is it okay to remove this logic? `BasicSupport.getLibertyDirectoryPropertyFiles()` requires the server directory to exist.

From what I can tell it is getting additional files to resolve server.xml `include` tags, but not relevant to feature generation as we are gathering from the source directory rather than server directory.

Reply:
Commented this code out for now and marked as a future TODO:
https://github.com/OpenLiberty/ci.maven/pull/1312/files#diff-f13f2e96a8e7ac60edcefdbaf97779e5f6926ac505277f90e8fa1d3b46507384R103-R113

Reply:
Could you open an issue to track this? I think for the final release we could implement a similar capability using the properties files in the `src/main/liberty` directory.

Reply:
Opened https://github.com/OpenLiberty/ci.maven/issues/1317